src/aics-image-viewer/components/ChannelsWidgetRow/index.tsx

```diff
@@ -51,14 +48,9 @@ const ChannelsWidgetRow: React.FC<ChannelsWidgetRowProps> = (props: ChannelsWidg
     changeChannelSetting(index, "isosurfaceEnabled", target.checked);
   };
 
-  const createChannelSettingHandler = <K extends ChannelStateKey>(settingKey: K) => {
-    return (newValue: ChannelState[K]) => changeChannelSetting(index, settingKey, newValue);
-  };
-
-  const _onIsovalueChange = createChannelSettingHandler("isovalue");
-  const onIsovalueChange = ([newValue]: number[]): void => _onIsovalueChange(newValue);
-  const _onOpacityChange = createChannelSettingHandler("opacity");
-  const onOpacityChange = ([newValue]: number[]): void => _onOpacityChange(newValue / ISOSURFACE_OPACITY_SLIDER_MAX);
+  const onIsovalueChange = ([newValue]: number[]): void => changeSettingForThisChannel("isovalue", newValue);
+  const onOpacityChange = ([newValue]: number[]): void =>
+    changeSettingForThisChannel("opacity", newValue / ISOSURFACE_OPACITY_SLIDER_MAX);
```

Comment:
should this replace all the other usages of `props.changeChannelSetting` in this component, lie the ones above for volumeEnabled and isosurfaceEnabled?